.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+**Issue #, if available:** n/a
+
+### Notes
+
+
+### Testing
+
+
+Before you submit a pull request, please make sure you have to following:
+
+- [ ] Your code should contain tests relevant for the problem you are solving
```

Comment:
Maybe reword this to: "Your code contains tests that cover all new code and changes"